dstep/Configuration.d

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright: Copyright (c) 2016 Jacob Carlborg. All rights reserved.
+ * Authors: Jacob Carlborg
+ * License: $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0)
+ */
+module dstep.Configuration;
+
+import dstep.translator.Options;
+
+/**
+    Aggregation of global configuration options affecting the program
+ */
+struct AppConfig
```

Comment:
* Please name this `Configuration`, or `Config`
* When there's a one to one mapping of a declaration and a module please use the same name for the declaration and the module